src/common/Header/index.js

```diff
@@ -28,8 +29,12 @@ const Header = () => {
                         <TitleText>Movies Browser</TitleText>
                     </TitleWrapper>
                     <ButtonsWrapper>
-                        <HeaderButton>MOVIES</HeaderButton>
-                        <HeaderButton>PEOPLE</HeaderButton>
+                        <StyledNavLink to="/movies">
```

Comment:
I think we dont need buttons for that.
How about we just use `<StyledNavLink to="/movies" /> ` with out button styles?